### PR TITLE
Redesign Refuelings Screen

### PR DESCRIPTION
**Pull request by baylessj**
Also part of #304.

Dependent on the Todos redesign going in first.

### CODE REVIEW

File: assets/intl/strings.json [hunk 1]
```diff
@@ -212,5 +212,26 @@
...
+    "many": "{{ count }} Late ToDos"
+  },
+  "due_soon_todo": {
+    "one": "{{ count }} ToDo Due Soon",
+    "many": "{{ count }} ToDos Due Soon"
+  },
+  "delete_todo": "Delete ToDo",
+  "due_by": "Due by",
+  "or": "or",
+  "sun_short": "Sun",
```
Comment (DavBfr) [inline assets/intl/strings.json:230]: This is already available as a flutter value, no need to translate it here

File: lib/screens/home/views/refuelings.dart [hunk 2]
```diff
@@ -1,61 +1,258 @@
...
+        child: Container(
+            padding: EdgeInsets.fromLTRB(10, 5, 10, 5),
+            child: Row(
+              mainAxisAlignment: MainAxisAlignment.spaceBetween,
+              crossAxisAlignment: CrossAxisAlignment.start,
+              children: List.generate(
+                  7, // days in a week
+                  (index) => _CalendarDay(
+                      name: weekdays[index],
+                      number: dayRange[index],
```
Comment (DavBfr) [inline lib/screens/home/views/refuelings.dart:147]: This has to be converted to a proper calendar day

File: assets/intl/strings.json (hunk 1 above)
Comment (baylessj) [reply assets/intl/strings.json]: Thanks for pointing that out, I didn't pick up on that when reading through the intl docs. Updated.

File: lib/screens/home/views/refuelings.dart (hunk 2 above)
Comment (baylessj) [reply lib/screens/home/views/refuelings.dart]: I'm not entirely clear on what you're mentioning here, but I _think_ I've fixed it with my latest commit. I hadn't handled the case when past days were part of the previous month, now the displayed dates match the calendar correctly.

**Comment by baylessj** [comment]
Review was dismissed automatically because I rebased, no changes were made to the code. Going ahead and merging as a result.

**Pull request merged**